packages/cozy-client/src/CozyClient.js

```diff
@@ -405,19 +405,27 @@ export default class CozyClient {
   }
 
   hydrateRelationships(document, relationships) {
-    const methods = this.getRelationshipStoreAccessors()
+    const methods = this.getRelationshipcessors()
```

Comment:
I think you didn't want to remove these characters ;)

Reply:
Eagle eye ! 🦅